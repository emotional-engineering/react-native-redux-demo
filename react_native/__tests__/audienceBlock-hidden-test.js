import React from 'react';
import MenuItem from '../elements/audienceBlock';

import renderer from 'react-test-renderer';

test('renders correctly', () => {

  const data = { };

  const tree = renderer.create(<audienceBlock visible={false} switchPlatformStat={()=>{}} />).toJSON();
  expect(tree).toMatchSnapshot();
});
