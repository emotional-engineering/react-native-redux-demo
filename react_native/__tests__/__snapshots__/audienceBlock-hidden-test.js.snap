// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<audienceBlock
  switchPlatformStat={[Function]}
  visible={false}
/>
`;
